sherpa/astro/plot.py

```diff
@@ -165,17 +162,17 @@ def prepare(self, data, stat=None):
         self.xlo, self.xhi = _check_hist_bins(self.xlo, self.xhi)
 
 
-class ModelPHAHistogram(HistogramPlot):
+class ModelPHAHistogram(shplot.HistogramPlot):
     """Plot a model for a PHA dataset.
 
     The filtering and grouping from the PHA datset
     are used to create the bins for the model.
     """
 
-    histo_prefs = backend.get_model_histo_defaults()
+    histo_prefs = shplot.backend.get_model_histo_defaults()
 
     def __init__(self):
-        HistogramPlot.__init__(self)
+        super().__init__()
```

Comment:
:+1: 

We have a lot of old code that doesn't use super() so it's nice to update them when we get the chance!